src/platforms/common/cdcacm.c

```diff
@@ -188,14 +188,14 @@ static const struct usb_endpoint_descriptor uart_data_endp[] = {{
 	.bDescriptorType = USB_DT_ENDPOINT,
 	.bEndpointAddress = 0x03,
 	.bmAttributes = USB_ENDPOINT_ATTR_BULK,
-	.wMaxPacketSize = CDCACM_PACKET_SIZE,
+	.wMaxPacketSize = 48, //CDCACM_PACKET_SIZE,
```

Comment:
Magic numbers are bad, and misleading comments are worse.  If this needs to change, perhaps we just change the value of `CDCACM_PACKET_SIZE`?